capstone-project/src/main/angular-webapp/src/app/app.module.ts

```diff
@@ -0,0 +1,30 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
+import { GoogleMapsModule } from '@angular/google-maps';
+import { MatSliderModule } from '@angular/material/slider';
+
```

Comment:
Remove empty line

Reply:
I saw in many angular examples that they separate  between imports of modules and imports of components, so I thought I'll stick to that convention (but I didn't find a reference to that in the style guide - https://angular.io/guide/styleguide)